README.md

```diff
@@ -66,139 +69,222 @@ or
   :load-path "/path/to/lib")
 ```
 
-## 📇 Commands
+## 🖥 Usage
+
+### 📇 Commands
 
-| Commands                   | Description                                                                 |
+The following are the functions provided by `ts-fold-mode`
+| Commands | Description |
 |----------------------------|-----------------------------------------------------------------------------|
-| `ts-fold-close`            | fold the current syntax node.                                               |
-| `ts-fold-open`             | open all folds inside the current syntax node.                              |
+| `ts-fold-close` | fold the current syntax node. |
+| `ts-fold-open` | open all folds inside the current syntax node. |
 | `ts-fold-open-recursively` | open the outmost fold of the current syntax node. Keep the sub-folds close. |
-| `ts-fold-close-all`        | close all foldable syntax nodes in the current buffer.                      |
-| `ts-fold-open-all`         | open all folded syntax nodes in the current buffer.                         |
-| `ts-fold-toggle`           | toggle the syntax node at `point'.                                          |
+| `ts-fold-close-all` | close all foldable syntax nodes in the current buffer. |
+| `ts-fold-open-all` | open all folded syntax nodes in the current buffer. |
+| `ts-fold-toggle` | toggle the syntax node at `point'. |
 
-## 🔨 Supported languages
+### 🔨 Supported languages
 
 > ⚠️ Please sort these two lists alphabetically!
 
 These languages are fairly complete:
 
-* Bash
-* C / C++ / C# / CSS
-* Elixir
-* Go
-* HTML
-* Java / JavaScript / JSX / JSON / Julia
-* Nix
-* PHP / Python
-* R / Ruby / Rust
-* Scala / Swift
-* TypeScript / TSX
+- Bash
+- C / C++ / C# / CSS
+- Elixir
+- Go
+- HTML
+- Java / JavaScript / JSX / JSON / Julia
+- Nix
+- PHP / Python
+- R / Ruby / Rust
+- Scala / Swift
+- TypeScript / TSX
 
 These languages are in development:
 
-* Agda
-* Elm
-* Emacs Lisp
-* OCaml
-* XML (upstream)
+- Agda
+- Elm
+- Emacs Lisp
+- OCaml
+- XML (upstream)
 
-## ⚖️ Indicators Mode
+## 📝 Customization
 
-<p align="center">
-<img src="./etc/indicators.png" width="40%" height=480%"/>
-</p>
+Although ts-fold aims to have good folding out of the box for all supported definitions, people will indubitably have their own preferences or desired functionality. The following section outlines how to add your own folding definitions and folding functions to make ts-fold work for you. If there are any improvements you find for existing or new languages, please do raise a PR so that others may benefit from better folding in the future!
 
-You need to load `ts-fold-indicators-mode`:
-- `use-package`
-   ```elisp
-   (use-package ts-fold-indicators
-   :straight (ts-fold-indicators :type git :host github :repo "emacs-tree-sitter/ts-fold"))
-   ```
+### :white_circle: Folding on new nodes
```

Comment:
I prefer unicode here -> ⚪.

Reply:
Added here and and moved the one other occurence of github styling to unicode.